packages/xdl/src/start/startAsync.ts

```diff
@@ -49,6 +49,13 @@ export async function startAsync(
   verbose: boolean = true
 ): Promise<ExpoConfig> {
   assertValidProjectRoot(projectRoot);
+  Analytics.unifiedClient.logEvent('dau', {
```

Comment:
I checked with Jon and this is how he wants it formatted. I think the reason is that the definition of what a dau is can change over time.  It can include events in other codebases and we want them all listed as a single 'dau' event.